labs-ios-starter/Login/View Controllers/LoginViewController.swift

```diff
@@ -61,13 +64,17 @@ class LoginViewController: DefaultViewController {
                 self.presentedViewController == nil else { return }
                 
             if exists {
-                self.performSegue(withIdentifier:
-                    .getSegueID(.showDetailProfileList), sender: nil)
+                self.handleLogin()
```

Comment:
Calling empty method?

Reply:
We can go over this tomorrow in standup if you'd like. I subclassed UIViewController (LoginViewController) and created that method. The class that inherits from LoginViewController (TopicsViewController) overrides the method and implements it. 

This was the cleanest way I could move everything around without spending a ton of time refactoring. It also gives us the flexibility of being able to make any ViewController the one that responds to login.